iocore/net/unit_tests/sni_conf_test.yaml

```diff
@@ -0,0 +1,5 @@
+sni:
```

Comment:
I think legal wants our copyrights on these yaml files, as annoying as that is.

The only files we don't add them are to things like gold files in which they can't be added.